### ISSUE TITLE
How To Bulk Edit - selecting the thing to replace

### ISSUE THREAD

**Issue by rosiel** [Islandora/documentation#1696]
We saw a demo of Views Bulk Edit today in the sandbox call.

Something we may want to do is to replace specific values with a different value.  once we've used our bulk-edit view to find and select the items with this specific field value, we are able to:

* Append a new field value to the objects
* Replace *all* of the existing values in the field with the ones we enter in the bulk edit interface.

What it does not allow is, for multi-value fields, replacing *only* the target value with a different one.

**Comment by elizoller** [comment]
that's dangerous

**Comment by rosiel** [comment]
Yes! But it is part of the process of "we migrated before we had cleaned up our data", and is also part of the work of having authorities. Is there an export-import process that would work for this?

**Comment by elizoller** [comment]
personally i'd probably do an export as CSV, clean up the data, and do a migration to update the records.

**Comment by mjordan** [comment]
I'd be curious to know if any metadata-focused system can do this using its standard metadata editing tools.

I would recommend @elizoller 's approach. I assume that Migrate lets you indicate that you're overwriting all values in a field with the incoming ones, but Islandora Workbench has an [open issue](https://github.com/mjordan/islandora_workbench/issues/47) about providing an "append", "delete" (all values in field), and "replace" update action, which I've been putting off to come after a [major refactoring of the code](https://github.com/mjordan/islandora_workbench/issues/72) that deals with different Drupal field types.
